src/main/java/org/launchcode/VennTime/controllers/HomeController.java

```diff
@@ -39,6 +42,15 @@ public String processCreateEvent(Model model, @ModelAttribute @Valid CreateEvent
             return "home";
         }
 
+        LocalTime startTime = createEventDTO.getStartTime();
+        LocalTime endTime = createEventDTO.getEndTime();
+
+        if (startTime.isAfter(endTime)) {
+            errors.rejectValue("startTime", "startTime.isAfterEndTime", "Please enter a valid start and end time.");
```

Comment:
Error message should be more descriptive, user ight not know why it's being rejected, esp. if it is somethin like: 

start: 1:00 pm, end 2:00 am